spec/models/mixins/miq_web_server_worker_mixin_spec.rb

```diff
@@ -68,6 +68,22 @@ def self.binding_address
       end
     end
 
-    expect(test_class.binding_address).to eq (before + "JUNK")
+    expect(test_class.binding_address).to eq(before + "JUNK")
+  end
+
+  describe '#rails_application' do
+    context 'websocket' do
+      let(:worker) { FactoryGirl.create(:miq_websocket_worker, :uri => 'http://127.0.0.1:3000') }
+      it 'creates a websocket server' do
+        expect(worker.rails_application).to be_a_kind_of(WebsocketServer)
+      end
+    end
+
+    context 'rails' do
+      let(:worker) { FactoryGirl.create(:miq_ui_worker, :uri => 'http://127.0.0.1:3000') }
+      it 'creates a rails server' do
+        expect(worker.rails_application).to be_a_kind_of(Vmdb::Application)
```

Comment:
We add this code above ^, so the first thought I have is we should demonstrate that this memoized value is not shared between instances from two different subclasses.    By collapsing the two tests into one, you do this and also test one specific thing: `rails_application`.